app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -236,6 +244,10 @@ class NewMessageFragment : Fragment() {
     }
 
     override fun onDestroyView() {
+        binding.editor.exportHtml { html ->
+            newMessageViewModel.editorBodyLoader.postValue(BodyContentPayload(html, BodyContentType.HTML_SANITIZED))
+        }
```

Comment:
As @tevincent also asked me, I see that this could be hard to understand. I will add a comment to follow his advice. This block of code is needed in order to keep and reload the content of the editor across configuration changes